doc/release_notes.rst

```diff
@@ -13,6 +13,7 @@ This part of documentation collects descriptive release notes to capture the mai
 
 **New Features and Major Changes**
 
+* Drop duplication of retrieve_data and COST_DIR, add params and update technology-data version `PR #1120 <https://github.com/pypsa-meets-earth/pypsa-earth/pull/1120>`__ and `PR #1120 <https://github.com/pypsa-meets-earth/pypsa-earth/pull/1249>`__
 
```

Comment:
May it be a good idea to add another release note mentioning the same PR to document also the changes in processing hydro profiles?

Reply:
Updated :)